src/elm/Page/Shop/Editor.elm

```diff
@@ -65,21 +60,6 @@ initUpdate productId loggedIn =
 
 
 
--- SUBSCRIPTIONS
-
-
-subscriptions : Model -> Sub Msg
-subscriptions model =
-    case maybeForm model of
-        Nothing ->
-            Sub.none
-
-        Just form ->
-            Sub.map PressedEnter (Events.onKeyDown decodeEnterKeyDown)
-                |> MarkdownEditor.withSubscription form.description
-
-
-
```

Comment:
Same thing is happening on this page, if I focus in, it gives an error. On `title` and `price`